src/DeviceInput/deviceInputSystem.ts

```diff
@@ -71,24 +92,22 @@ export class DeviceInputSystem implements IDisposable {
         const device = this._inputs[deviceName];
 
         if (!device) {
-            throw `Unable to find device ${deviceName}`;
+            return null;

```

Comment:
It's not clear to me why this needs to change.

Reply:
Originally, this was changed to account for checking for input before a device exists but there are checks outside of this so I put the throw back as it just never throw under *normal* circumstances.